### ISSUE TITLE
Does LSC support high availability?

### ISSUE THREAD

**Issue by sht33** [lsc-project/lsc#309]
Does LSC support high availability, such as master-master mode or master-slave mode?

**Comment by coudot** [comment]
You can install a connector on two nodes and have a cron that runs it on both nodes (but not at the same time)

**Comment by sht33** [comment]
> You can install a connector on two nodes and have a cron that runs it on both nodes (but not at the same time)

I have two nodes, Node 1 and Node 2, both of which have LDAP installed and are set up as master and slave. Node 1 also has LSC installed to synchronize LDAP data to the AD domain. Now I want to make LSC highly available. Do I just need to install an LSC on Node 2?

**Comment by coudot** [comment]
Yes, but if your master is not available, LSC will not be able to write in the slave, unless you promote slave into master.

**Comment by sht33** [comment]
> Yes, but if your master is not available, LSC will not be able to write in the slave, unless you promote slave into master.

How to avoid synchronizing duplicate data from ldap to ad domain

**Comment by coudot** [comment]
LSC will always compare data before modifying them.

Just run LSC at different hours on each node, depending on the frequency you desire

**Comment by sht33** [comment]
> LSC 在修改数据之前总是会进行比较。
> 
> 只需在每个节点上根据所需的频率在不同的时间运行 LSC

Thank you so much

**Comment by sht33** [comment]
> LSC will always compare data before modifying them.
> 
> Just run LSC at different hours on each node, depending on the frequency you desire

How to set different running frequencies on different nodes？Is it set in the lsc.xml configuration file? Or is it set using crontab -e?

**Comment by soisik** [comment]
Hello @sht33,

The frequencies of you task is indeed usually set using crontab -e -u lsc, so you just have to pick a different time for the second node.
